omero/sysadmins/unix/server-centos6-ice36.txt

```diff
@@ -147,7 +147,6 @@ Nginx configuration directory, and disable the default configuration:
     :end-before: #end-nginx
 
 
-
 Running OMERO
```

Comment:
Having 'Running OMERO' right underneath '...Running OMERO.web' is slightly confusing, maybe this would be better as 'Running OMERO.server'?